Packs/Reco/Integrations/Reco/Reco.py

```diff
@@ -1251,6 +1272,10 @@ def main() -> None:
                                                        demisto.args()["domain"],
                                                        int(demisto.args()["last_interaction_time_in_days"]))
             return_results(result)
+        elif command == "reco-add-exclusion-filter":
+            result = add_exclusion_filter(reco_client, demisto.args()["key_to_add"],
+                                          demisto.args()["values_to_add"].split(','))
```

Comment:
You can use `argToList`
```suggestion
                                          argToList(demisto.args()["values_to_add"]))
```